packages/api-server/src/convert-tx.ts

```diff
@@ -347,3 +404,38 @@ function publicKeyToEthAddress(publicKey: HexString): HexString {
       .toString("hex");
   return ethAddress;
 }
+
+export interface AutoCreateAccountCacheValue {
+  tx: HexString;
+  fromAddress: HexString;
+}
```

Comment:
It seems `AutoCreateAccountCacheValue` is a cache-use type. Should we move it into `cache/types.ts`?

Reply:
Updated